src/cunumeric/unary/unary_red.cc

```diff
@@ -74,19 +61,6 @@ struct ArgRedImplBody<VariantKind::CPU, OP_CODE, CODE, DIM> {
       lhs.reduce(point, LHS(point[collapsed_dim], rhs[point]));
     }
   }
-
-  void operator()(AccessorRW<LHS, DIM> lhs,
-                  AccessorRO<RHS, DIM> rhs,
-                  const Rect<DIM>& rect,
-                  const Pitches<DIM - 1>& pitches,
-                  int collapsed_dim,
-                  size_t volume) const
-  {
-    for (size_t idx = 0; idx < volume; ++idx) {
-      auto point = pitches.unflatten(idx, rect.lo);
-      OP::template fold<true>(lhs[point], LHS(point[collapsed_dim], rhs[point]));
-    }
-  }
 };
```

Comment:
It looks like `ArgRedImplBody` is no longer used, and can be removed from this file and `unary_red_template.inl`.

Reply:
3f2769dd3e95aa4ad078d1a8c0db475038068f91